packages/@aws-cdk/aws-eks/lib/cluster-resource-handler/cluster.ts

```diff
@@ -198,7 +198,7 @@ export class ClusterResourceHandler extends ResourceHandler {
           Endpoint: cluster.endpoint,
           Arn: cluster.arn,
           CertificateAuthorityData: cluster.certificateAuthority?.data,
-          OpenIdConnectIssuerUrl: cluster.identity?.oidc?.issuer,
+          OpenIdConnectIssuerUrl: cluster.identity?.oidc?.issuer?.substring(8), // Strips off https:// from the issuer url
```

Comment:
This is technically a breaking change. Perhaps the correct approach is to add another attribute `OpenIdConnectIssuer` which will return the value without the `https://` prefix.

Reply:
Ah yeah. Good one. I'll make the change.

Reply:
Should I create the additional property on the Cluster construct as well?

Reply:
Sure